tasks/gen/component.cr

```diff
@@ -37,9 +49,13 @@ class Gen::Component < LuckyTask::Task
   end
 
   private def missing_name_error
-    if ARGV.first?.nil?
-      "Component name is required."
-    end
+    # Doing this because `component_class` will raise an exception if the value is missing
+    # but the error message would say "component_class is missing" which isn't as nice of
+    # an error message. This lets the UI remain the same until this whole deal can be refactored
+    component_class
+    nil
+  rescue
+    "Component name is required."
```

Comment:
This is mainly just to keep the output the same as it is now. I have an issue on LuckyTask https://github.com/luckyframework/lucky_task/issues/8 so we can address this later.